Description: Look in the correct place for cddlib's header file.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-06-04

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -1038,7 +1038,7 @@
 then AC_LANG(C)
      AC_CHECK_HEADER(cdd.h,,
      	[ 
-	  CPPFLAGS="-I/usr/include/cddlib $CPPFLAGS"
+	  CPPFLAGS="-I/usr/include/cdd $CPPFLAGS"
 	  unset ac_cv_header_cdd_h
 	  AC_CHECK_HEADER(cdd.h,,BUILD_cddlib=yes,[#include <setoper.h>])
 	],
